[TW-1496] Migrate troubleshooting FAQ to LC (#5219)
postmanlabs/postman-docs: src/pages/docs/live-insights/live-insights-troubleshoot.md

#### Check the Live Collections diagnostics

Issues with traffic processing also show up in the Live Collections diagnostics dashboard. To access the diagnostics dashboard:

1. Select the live collection.
2. In the right-hand sidebar, select the Live Collections icon <img alt="Live Collections icon" src="https://assets.postman.com/postman-docs/v10/icon-live-collections.jpg#icon" width="16px"> next to the collection.
3. Select **View Diagnostics**.

### What do I do if my traffic is encrypted?

If your model is missing endpoints that you expect to be there, or is completely empty, it may be because you are trying to observe your API at a point where it's encrypted. In Postman, visit the Live Collections diagnostics dashboard and select one of the client reports listed there. To access the diagnostics dashboard:

1. Select the live collection.
2. In the right-hand sidebar, select the Live Collections icon <img alt="Live Collections icon" src="https://assets.postman.com/postman-docs/v10/icon-live-collections.jpg#icon" width="16px"> next to the collection.
3. Select **View Diagnostics**.

You'll see whether the LCA saw any traffic encrypted with TLS.

> If you’re running Kubernetes, follow the [installation instructions](/docs/live-insights/live-insights-gs) to install LCA as a sidecar. This enables the LCA to see encrypted traffic in most common Kubernetes deployments.

Reviewer: ```suggestion
If you’re running Kubernetes, follow the [installation instructions](/docs/live-insights/live-insights-gs) to install LCA as a sidecar. This enables the LCA to see encrypted traffic if a service mesh such as Istio is being used.
```
